alert.go

```diff
@@ -907,6 +913,14 @@ func (a *AlertNode) handleAlerta(alerta alertaHandler, ad *AlertData) {
 	resource := buf.String()
 	buf.Reset()
 
+	err = alerta.eventTmpl.Execute(&buf, ad.info)
```

Comment:
Right here you can create a new type like:

```go
type eventData struct {
      idInfo
      ID string
}
data := eventData{
     idInfo: ad.info.messageInfo.idInfo,
     ID: ad.ID,
}
err = alerta.eventTmpl.Execute(&buf, data)
```

This way on the ID Info data can be used in addition to the ID itself.